src/components/hand-controls.js

```diff
@@ -74,34 +126,120 @@ module.exports.Component = registerComponent('hand-controls', {
-        evt = isPressed ? 'pointup' : 'pointdown';
-        el.emit(evt);
+        if (evt.indexOf('touch') === 0) {
+          if (isTouched === this.triggerTouched) { return; }
+          this.triggerTouched = isTouched;
+        } else {
+          if (isPressed === this.triggerPressed) { return; }
+          this.triggerPressed = isPressed;
+        }
         break;
       case 'grip':
-        if (isPressed === this.gripPressed) { return; }
-        this.gripPressed = isPressed;
-        this.playAnimation('close', !isPressed);
-        evt = isPressed ? 'gripclose' : 'gripopen';
-        el.emit(evt);
+        if (evt.indexOf('touch') === 0) {
+          if (isTouched === this.gripTouched) { return; }
+          this.gripTouched = isTouched;
+        } else {
+          if (isPressed === this.gripPressed) { return; }
+          this.gripPressed = isPressed;
+        }
+        break;
+      case 'thumbstick':
+        if (isPressed === this.thumbstickPressed) { return; }
+        this.thumbstickPressed = isPressed;
+        shouldAnimate = false;
+        break;
+      case 'AorX':
+        if (isTouched === this.AorXTouched) { return; }
+        this.AorXTouched = isTouched;
+        break;
+      case 'menu':
+        if (isTouched === this.menuTouched) { return; }
+        this.menuTouched = isTouched;
+        break;
+      case 'surface':
+        if (isTouched === this.surfaceTouched) { return; }
+        this.surfaceTouched = isTouched;
         break;
     }
+    if (shouldAnimate) { this.animate(); }
   },
 
-  /**
+  animate: function () {
+    var animation;
+    var reverse = false;
+    if (this.gripPressed) {
+      if (this.surfacePressed || this.surfaceTouched ||
+          this.menuTouched || this.AorXTouched ||
+          this.trackpadPressed || this.trackpadTouched) {
+        if (!this.triggerPressed && !this.triggerTouched) {
+          // point
+          animation = 'pointing';
+        } else {
+          // make a fist
+          animation = 'press';
+        }
+      } else {
+        if (!this.triggerPressed && !this.triggerTouched) {
+          // pistol pose
+          animation = 'pistol';
+        } else {
+          // thumbs up
+          animation = 'thumb';
+        }
+      }
+    } else {
+      // grip not pressed
+      if (!this.triggerPressed && !this.triggerTouched) {
+        // TODO: seems as though we should have some additional poses here
+        animation = 'touch';
+        reverse = true;
+      } else {
+        // touch pose (?)
+        animation = 'touch';
+      }
+    }
+    this.playAnimation(animation, reverse);
+  },
+
+  // map to old vive-specific event names for now
+  animationEventMapping: {
+    'press': 'grip',  // e.g. grip button down
+    'touch': 'point', // e.g. trigger button down
+    'thumb': 'thumb'  // e.g. thumbs up pose - grip button down, trackpad / surface buttons up
+  },
+
+  emitAnimationEvents: function (animation, reverse) {
```

Comment:
The tracked-controls example did not take advantage of take gestures. It was wired so any button triggered a grab for simplicity since that demo did not have any other interaction besides grabbing cubes.